packages/app/pages/schedule.tsx

```diff
@@ -19,15 +30,18 @@ export default function Schedule() {
 
   const myEvents =
     data?.officeHours.map((e) => ({
-      title: e.title,
       start: e.startTime,
       end: e.endTime,
     })) ?? [];
-
+  // TODO: frontend guru pls cleanup height <3, month view is crunked :)
+  // esp on mobile
```

Comment:
I found a solution for the month view, turns out your calendar needs to have a set height in order for it to display properly. I just pushed a commit with my change for this.

Regarding responsiveness, it seems like the crunkiness is coming from the Toolbar. It's probably worth looking into a custom Toolbar if you can't find a built in responsive solution in the library: https://jquense.github.io/react-big-calendar/examples/index.html#prop-components. This seems like a good example of a custom Toolbar: https://github.com/jquense/react-big-calendar/issues/191.